jest.config.json

```diff
@@ -5,11 +5,8 @@
     "testEnvironment": "node",
     "transform": {},
     "setupFiles": [
-        "./test/jest.setup-file.js"
+        "./lib/test/jest.setup-file.js"
```

Comment:
nits: might want to use `ts-node` or `ts-jest` instead

Reply:
ts-node sometimes hide errors triggered by the `tsc` compiler, and one other dependency to track